packages/server/integrations/gitlab/GitLabAuthorizationManager.ts

```diff
@@ -0,0 +1,69 @@
+import {
```

Comment:
I was hoping to create a more generic authorization method that could live as a mixin or within a super as I noticed our oauth2 authorization code was _very_ repetitive

Reply:
This is still in the works